go/registry/api/status.go

```diff
@@ -0,0 +1,21 @@
+package api
+
+import (
+	epochtime "github.com/oasislabs/oasis-core/go/epochtime/api"
+)
+
+// FreezeForever is an epoch that can be used to freeze a node for
+// all (practical) time.
+const FreezeForever epochtime.EpochTime = 0xffffffffffffffff
+
+// NodeStatus is live status of a node.
+type NodeStatus struct {
+	// FreezeEndTime is the epoch when a frozen node becomes unfrozen.
```

Comment:
Slightly misleading, given the check.

Reply:
Yes once this passes the node also needs to explicitly unfreeze itself to be unfrozen (at which point this will be reset). I joined the two under a single value, could add a separate flag for being frozen.

Reply:
I think them being merged is fine, maybe clarify the doc comment.